tests/installation/addon_products_sle.pm

```diff
@@ -71,6 +68,9 @@ sub handle_all_packages_medium {
         push @addons, $i if !grep(/^$i$/, @addons);
     }
 
+    # Add python2 module, refer to https://jira.suse.de/browse/SLE-3167
```

Comment:
If SLE15SP2 always add python2 automatically, we can just don't add python2 when target is SLE15SP2. WDYT?

Reply:
Please check the background.